packages/api-server/src/plugins/services/explorer-service/index.ts

```diff
@@ -149,7 +149,7 @@ export class ExplorerService {
         const questionId = randomUUID();
         const logger = this.logger.child({ questionId: questionId });
         const normalizedQuestion = this.normalizeQuestion(q);
-        if (normalizedQuestion.length > 255) {
+        if (normalizedQuestion.length > 384) {
```

Comment:
Why is 384? Should we enlarge it to 512?

Reply:
512 is fine

Reply:
Done